pkgs/tools/misc/nautilus-open-any-terminal/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, pkg-config
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, gnome
+, gobject-introspection
+, gsettings-desktop-schemas
+, gtk3
+, python3
```

Comment:
```suggestion
```

Reply:
Why not drop `python3Packages`. That one is actually an alias.